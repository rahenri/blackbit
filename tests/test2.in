new
protover
sd 10
post
e2e4
quit
