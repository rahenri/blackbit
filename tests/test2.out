feature myname="raphael-chess8" ping=1 usermove=1 draw=0 variants="normal" sigint=0 sigterm=0 setboard=1 playother=1 analyze=0 colors=0 done=1
10 -120 0 54601637 e7e5 d1h5 b8c6 f1c4 d8e7 b1c3 c6d4 h5d1 e7g5 g2g4
move e7e5
